src/App.js

```diff
@@ -62,15 +63,21 @@ const PickupCard = ({ field, day, time, address, contact }) => {
 }
 
 const Footer = () => {
-    return <footer className='footer'>
-        <div className='has-text-centered'>
+    return <footer className='footer level'>
+        <div className='level-item has-text-centered'>
             <p>
                 <strong>Made by </strong>
                 <a href='https://linkedin.com/in/oludavid' target='_blank' rel="noopener noreferrer nofollow">
                     <span className='has-text-success'>Olu</span>
                 </a>
             </p>
         </div>
+        <div className='level-item has-text-centered'>
+            <a href="https://www.facebook.com/groups/denverpickupsoccer/?__cft__[0]=AZV9jHuReRSAM-1Lkx50LDVO_vG2l3k4ietNdFSJGJEmQDOV6U3uSZRlmobsUQjOCaFj9_BjxbtXM1K734fDT5Us8DyZcjgJ1U3c17g8q2lIkvT2qyTjvW1K7Vx-ecMPk2kfttwQQiG3W_qYIRz7i_LXD5Lb9Z5pqWFbktUGjmA7o_2hCXHrYW3ThbmWv_IkOOA&amp;__tn__=-UC%2CP-R">
+                For More Games Follow: Denver Pick-Up Soccer 
+                <img className="svg_icon" src={fb} alt='Facbook Icon' />
```

Comment:
`svg_icon` sounds generic. Could you make the class name specifically for this Facebook icon since that's what the styling specifically applies to.

If in the future, we're repeating styling for multiple icons, then we can update to a generic class name.
```suggestion
                <img className="facebook_icon" src={fb} alt='Facebook Icon' />
```